sway/desktop/output.c

```diff
@@ -426,7 +426,7 @@ static void send_frame_done_iterator(struct sway_output *output, struct sway_vie
 	int delay = data->msec_until_refresh - output->max_render_time
 			- view_max_render_time;
 
-	if (output->max_render_time == 0 || view_max_render_time == 0 || delay < 1) {
+	if (delay == data->msec_until_refresh || delay < 1) {
```

Comment:
Hmm. Would the code be clearer by replacing `delay == data->msec_until_refresh` with `(output->max_render_time == 0 && view_max_render_time == 0)`?